karma.conf.js

```diff
@@ -15,6 +15,7 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
+      'node_modules/babel-polyfill/dist/polyfill.min.js',
```

Comment:
Have you tested this on Internet Explorer? We have this polyfill in GC but EDGE is still missing promises and a few array function ( .includes .indexOf ). So annoying.